archetypes/data_store.xml

```diff
@@ -0,0 +1,49 @@
+<ArchetypeLibrary
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://www.example.org/ArchetypeLibrary" 
+    xsi:schemaLocation="http://www.example.org/ArchetypeLibrary ArchetypeLibrarySchema.xsd">
+
+	<FunctionArchetype name="Data_Store">

```

Comment:
Would it be possible to use the archetype in at least one EXISTING test model? E.g. in AUR-CBI-MDL-0090, CBI Reference Component Set shall include a DataStore component

Reply:
Done